messaging-modules/spring-apache-camel/pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-spring-javaconfig</artifactId>

```

Comment:
It seems we can fully remove the camel-spring-javaconfig, and instead use spring boot
https://stackoverflow.com/questions/73391193/camelconfiguration-deprecated-and-removed

So, remove this and adjust ContentBasedFileRouterConfig class (remove extends and @overrides)

Reply:
I did the suggested adjustment